spec/api/documentation/service_keys_api_spec.rb

```diff
@@ -5,13 +5,18 @@
   let(:admin_auth_header) { admin_headers['HTTP_AUTHORIZATION'] }
   let(:service_instance) { VCAP::CloudController::ManagedServiceInstance.make }
   let(:service_key) { VCAP::CloudController::ServiceKey.make }
+  let(:guid) { service_key.guid }
   authenticated_request
 
   before do
     create_response_body = { 'credentials' => service_instance.credentials }
     stub_bind(service_instance, body: create_response_body.to_json)
+    stub_unbind(service_key)
   end
 
+  standard_model_list :service_key, VCAP::CloudController::ServiceKeysController
+  standard_model_delete :service_key, async: false
+
   post '/v2/service_keys' do
```

Comment:
the standard_model_delete method will test the delete function of the model, so this will cover the delete API doc